app/src/main/java/com/android/brewr/ui/explore/CoffeeInformationCard.kt

```diff
@@ -61,7 +61,7 @@ fun CoffeeInformationCardScreen(coffee: Coffee, onClick: () -> Unit) {
                   buildAnnotatedString1 {
                     append("Address: ")
                     addStyle(SpanStyle(fontWeight = FontWeight.Bold), 0, "Address: ".length)
-                    append(coffee.location.address)
+                    append(coffee.location.name)
```

Comment:
Is there a certain reason to rename the "address" to "name" ? Just asking by curiosity. You are currently calling "Address" as parameter here : addStyle(SpanStyle(fontWeight = FontWeight.Bold), 0, "Address: ".length) whereas you append coffee.location."name".  Before's name is more coherent no ?

Reply:
This is because I removed the duplicate of Location.kt. Maybe we should fix the name of the Location parameter "name" to make it more explicit